tests/e2e/isbservice.go

```diff
@@ -347,36 +348,39 @@ func updateISBServiceRollout(
 	originalPipelineName, err := getPipelineName(Namespace, pipelineRolloutName)
 	Expect(err).ShouldNot(HaveOccurred())
 
-	updateISBServiceRolloutInK8S(isbServiceRolloutName, func(rollout apiv1.ISBServiceRollout) (apiv1.ISBServiceRollout, error) {
+	UpdateISBServiceRolloutInK8S(isbServiceRolloutName, func(rollout apiv1.ISBServiceRollout) (apiv1.ISBServiceRollout, error) {
 		rollout.Spec.InterStepBufferService.Spec.Raw = rawSpec
 		return rollout, nil
 	})
 
 	// both "dataLoss" fields and "recreate" fields have risk of data loss
 	dataLossRisk := dataLossFieldChanged || recreateFieldChanged
-
-	if upgradeStrategy == config.PPNDStrategyID {
+	if UpgradeStrategy == config.PPNDStrategyID {
 		if dataLossRisk {
 
-			document("Verify that in-progress-strategy gets set to PPND")
+			Document("Verify that in-progress-strategy gets set to PPND")
 			verifyInProgressStrategyISBService(Namespace, isbServiceRolloutName, apiv1.UpgradeStrategyPPND)
-			verifyInProgressStrategy(pipelineRolloutName, apiv1.UpgradeStrategyPPND)
-			verifyPipelinePaused(Namespace, pipelineRolloutName)
+			// if we expect the pipeline to be failed?
+			// short term workaround
```

Comment:
maybe we should just pass in a boolean to the function called `pipelineIsFailed`?

Reply:
That would probably be best, just didn't want to continually increase the function signature but this would be more helpful.